docs/Users_Guide/difficulty_index.rst

```diff
@@ -0,0 +1,173 @@
+
+.. code-block:: ini
+
+    cp $METPLOTPY_BASE/test/difficulty_index/swh_North_Pacific_5dy_ensemble.npz $WORKING_DIR
+
+Where *$METPLOTPY_BASE* is the directory where you saved the METplotpy source code and *$WORKING_DIR* is the
+directory you created to store input data.
+
+
+Required Packages
+_________________
+
+The following Python packages are necessary:
+
+- Python 3.6
+- numpy
+- matplotlib 2.2.2 (minimum)
+- scipy
+
+
+Configuration Files
+___________________
+
+All the settings for the example difficulty index plot are incorporated in
+the mycolormaps.py and plot_difficulty_index.py code.  The example_difficulty_index.py script imports these modules to
+create six sample plots.  The location of where these plots are saved are determined by settings in the
+example_difficulty_index.yaml configuration file:
+
+.. literalinclude:: ../../test/difficulty_index/example_difficulty_index.yaml
+
+
+Copy this config file from the directory where the source code was
+saved to the working directory:
+
+.. code-block:: ini
+
+  cp $METPLOTPY_BASE/test/difficulty_index/example_difficulty_index.yaml $WORKING_DIR/example_difficulty_index.yaml
+
+
+Modify the *input_filename* setting in the
+*$METPLOTPY_BASE/test/difficulty_index/example_difficulty_index.yaml* file to
+explicitly point to the *$METPLOTPY_BASE/test/difficulty_index*
+directory (where the config file and sample data reside).
+Replace the path */path/to/swh_North_Pacific_5dy_ensemble.npz* with the full path
+*$METPLOTPY_BASE/test/difficulty_index/swh_North_Pacific_5dy_ensemble.npz*
+(including replacing *$METPLOTPY_BASE* with the full path to the METplotpy
+installation on the system).  Modify the *stat_fig_basename*
+setting to point to the output path where the two statistics plots will be saved.
+These plots will have a prefix of *swh_North_Pacific_5dy_*
+Modify the *diff_fig_basename*
+setting to point to the output path where the other four difficulty index plots will be saved.
+These plots will have a prefix of *swh_North_Pacific_difficulty_index_*
+
+
+
+METplus Configuration
+=====================
+
+Run from the Command Line
+=========================
+
+To generate the sample difficulty index plots, perform the following:
+
+*  If using the conda environment, verify the conda environment
+   is running and has has the required Python packages outlined in the **Required Packages** section above.
+
+
+Where $METPLOTPY_BASE is the directory where you saved the METplotpy source code and $WORKING_DIR is the directory
+you created earlier to store the sample input data.
+
+* Change directory to the $WORKING_DIR:
+
+    .. code-block:: ini
+
+      cd $WORKING_DIR
+
+where *$WORKING_DIR* is the directory where you copied all the necessary files (e.g. /home/users/someuser/working_dir).
+
+
+* Run the following on the command line:
+
+  .. code-block:: ini
+
+    python example_difficulty_index.py example_difficulty_index.yaml
```

Comment:
```suggestion
    python ${METPLOTPY_BASE}/test/difficulty_index/example_difficulty_index.py --config example_difficulty_index.yaml
```
The command that is currently listed did not work for me, but the one I am suggesting did.